Berksfile

```diff
@@ -3,10 +3,8 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'apt', '~> 3.0.0'
-  cookbook 'yum', '~> 3.10.0'
-  cookbook 'zypper', '~> 0.2.1'
-  cookbook 'pacman', '~> 1.1.1'
```

Comment:
At this point, I do not think it makes sense to remove arch support as it was added. I guess we can reach out to the original contributor and get their feedback. We do want to make sure we are using the latest, it is the same reason apt, yum are there.